packages/vue/src/components/search/DataSearch.jsx

```diff
@@ -262,15 +264,31 @@ const DataSearch = {
 		},
 	},
 	methods: {
+		validateDataField() {
+			const propName = 'dataField';
+			const componentName = 'DataSearch';
```

Comment:
@ShahAnuj2610 Use it from `name` property.
https://github.com/appbaseio/reactivesearch/blob/bb371f05df1dce890f0b1d6bac6a6288922defae/packages/vue/src/components/search/DataSearch.jsx#L54

Reply:
done

Reply:
@ShahAnuj2610 How is it done?
You're still hardcoding the `componentName`, it should be `this.name`.

Reply:
@bietkul check this https://github.com/appbaseio/reactivesearch/pull/1448/files#diff-29c2691a20b54ace57eca812d2d9ca86R269